src/main/java/com/example/damian/iracki/ticketBookingApp/services/TicketService.java

```diff
@@ -0,0 +1,34 @@
+package com.example.damian.iracki.ticketBookingApp.services;
+
+
+import com.example.damian.iracki.ticketBookingApp.entities.Ticket;
+import com.example.damian.iracki.ticketBookingApp.repositories.TicketRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TicketService {
+
+    private final TicketRepository ticketRepository;
+
+    @Autowired
+    public TicketService(TicketRepository ticketRepository) {
+        this.ticketRepository = ticketRepository;
+    }
+
+    public void addNewTicket(Ticket ticket){
+        Optional<Ticket> ticketOptional = ticketRepository.findTicketByScreeningIdAndNumberOfRowAndNumberOfSeatInRow
```

Comment:
Myślisz, że to dobrze, że w naszym kodzie domenowym używamy nazw funkcji wynikających z konwencji Springowej?

Reply:
Jeżeli w przyszłości chcielibyśmy zmienić framework to niedobrze.